lightwood/mixer/neural.py

```diff
@@ -289,7 +289,9 @@ def fit(self, train_data: EncodedDs, dev_data: EncodedDs) -> None:
 
         if self.fit_on_dev:
             self.partial_fit(dev_data, train_data)
-        self._final_tuning(dev_data)
+
+        if not self.timeseries_settings.is_timeseries:
+            self._final_tuning(dev_data)
```

Comment:
Wait... why are we not doing final tuning for timeseries? Why is the change part of this PR?

Reply:
It's on the PR description:

> Finally, and as an aside, Neural() now does not perform a final tuning on time series tasks (was incorrectly doing so without dependency data being passed).

I guess it could be a separate PR, if you prefer that.

Reply:
My bad... but shouldn't there be a PR to fix final tuning then?